rayon-threadlimit/src/thread_count.rs

```diff
@@ -0,0 +1,15 @@
+use std::sync::RwLock;
+
+//TODO remove this hack when rayon fixes itself
+lazy_static! {
+    static ref MAX_RAYON_THREADS: RwLock<usize> =
+        RwLock::new(sys_info::cpu_num().unwrap() as usize);
+}
+
+pub fn get_thread_count() -> usize {
+    *MAX_RAYON_THREADS.read().unwrap()
+}
+
+pub fn init_test_thread_count() {
```

Comment:
nit: how about:
```rust
pub fn set_thread_count(count: usize) ..
```

then do `set_thread_count(1)` from the one place that `init_test_thread_count()` is called instead

Reply:
Yeah I was just closing it off. Didn't want it to be an accessible api. But a hack's a hack. I can change it to this.

Reply:
I don't feel strongly :)